src/components/MainContainer.js

```diff
@@ -76,10 +78,20 @@ class MainContainer extends Component {
         this.setState({ mapContainerW, mapContainerH });
     }
 
-    handleCountySelect = (geoid) => {
-        fetchJSON(geoid)
-            .then(dataset => this.setState({ dataset, geoid }))
-            .catch(e => console.log('Fetch was problematic: ' + e.message));
+    async handleCountySelect(geoid) {
```

Comment:
`async handleCountySelect(geoid) {` -> `handleCountySelect = async (geoid) => {`

`handleCountySelect` needs to be binded or be an arrow function to have access to `this`

Reply:
Thank you @BenjiLee!